.prettierignore

```diff
@@ -0,0 +1,13 @@
+node_modules
+# Ignore artifacts:
+build
```

Comment:
I think it will also be necessary to ignore the `platforms` directory, and maybe `www/dist`

Is there no way to target Prettier at *only* the `www` folder?

Reply:
`/manual_lib` and `/json` must be ignored as well

Reply:
Thanks all of your advices!

I found the pattern only check `www` directory without `dist`, `manual_lib`, and `json`.
Can you please confirm this? 🙏

Reply:
If the strategy is to ignore everything except `www`, you can remove `node_modules` and the artifacts since these are already ignored by the `/*` rule

Reply:
Oh, I missed that! Thanks :) I deleted the line!